x-pack/plugins/observability/server/services/slo/find_slo_groups.ts

```diff
@@ -4,15 +4,26 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { FindSLOGroupsParams, FindSLOGroupsResponse, Pagination } from '@kbn/slo-schema';
-import { ElasticsearchClient } from '@kbn/core/server';
-import { findSLOGroupsResponseSchema } from '@kbn/slo-schema';
-import { Logger } from '@kbn/core/server';
-import { typedSearch } from '../../utils/queries';
+import {
+  FindSLOGroupsParams,
+  FindSLOGroupsResponse,
+  findSLOGroupsResponseSchema,
+  Pagination,
+} from '@kbn/slo-schema';
+import { ElasticsearchClient, Logger, SavedObject } from '@kbn/core/server';
+import { SavedQueryAttributes } from '@kbn/data-plugin/common';
+import {
+  AggregationsAggregationContainer,
+  QueryDslQueryContainer,
+  Sort,
+} from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { SubAggregateOf } from '@kbn/es-types/src';
```

Comment:
Missed this initially, please export from the top level dir sot that we can do

```suggestion
import { SubAggregateOf } from '@kbn/es-types';
```